### ISSUE TITLE
docblock error

### ISSUE THREAD

**Issue by Versud** [westacks/telebot#21]
```php
    WeStacks\TeleBot\BotManager

    ....

   /**
     * Get bot by name.
     * @param string $name bot name
     * @throws TeleBotObjectException
     * @return null|TeleBot
     */
    public function bot(string $name = null)
    {
        $bot = $name ?? $this->default ?? null;

        if (is_null($bot)) {
            throw TeleBotObjectException::defaultBotIsNotSet();
        }

        if (!isset($this->bots[$bot])) {
            throw TeleBotObjectException::botNotFound($bot);
        }

        if (!($this->bots[$bot] instanceof TeleBot)) {
            $this->bots[$bot] = new TeleBot($this->bots[$bot]);
        }

        return $this->bots[$bot];
    }
```

hello, how can this method return null?

**Comment by punyflash** [comment]
Hello! It can't return null, you are right.
I'll fix this on some bigger update or let someone make pull request as this issue is not even breaking autocompletion for IDEs.

**Comment by punyflash** [comment]
Fixed in 35dad59c2e34fce3d010d6e9f7ee42bb1d89387e
